libavfilter/dnn_filter_common.h

```diff
@@ -46,7 +45,6 @@ typedef struct DnnContext {
     { "output",             "output name of the model",   OFFSET(model_outputnames_string), AV_OPT_TYPE_STRING, { .str = NULL }, 0, 0, FLAGS },\
     { "backend_configs",    "backend configs",            OFFSET(backend_options),  AV_OPT_TYPE_STRING,    { .str = NULL }, 0, 0, FLAGS },\
     { "options", "backend configs (deprecated, use backend_configs)", OFFSET(backend_options),  AV_OPT_TYPE_STRING, { .str = NULL }, 0, 0, FLAGS | AV_OPT_FLAG_DEPRECATED},\
-    { "async",              "use DNN async inference",    OFFSET(async),            AV_OPT_TYPE_BOOL,      { .i64 = 1},     0, 1, FLAGS},
```

Comment:
do not remove it for backward compatible, just add deprecated like the above line.

Reply:
I mean comment updated in the second column, "use DNN async inference"

Reply:
Like this "use DNN async inference (ignored, use backend_configs='async=1')"?

Reply:
yes, otherwise, people will see program reports error and ends due to the unknown option, if he used async in the command line.

Reply:
Done. I had a question - how will people who haven't seen the source code see this comment? Is there any help command for the filter?

Reply:
yes, it is not complete backward compatible.

ffmpeg -h filter=<name of filter>

Reply:
I was thinking that we can make it backward compatible by passing this async parameter to the backend load function. 

Since this `async` is present in `DnnContext`, we can pass it to the load model functions and check if the user has set the value there. (in this case, we'd have to set a default value other than 0 or 1)

The only problem with this method is that if the user sets both the values `async=0` and `backend_configs=async=1`, we'll not know which one user was trying to use.

Reply:
i guess not many people are using the filter option, and such  backward compatible is too complex ...

Reply:
Okay. I have changed the comment in column 2.